src/Core/Data/Math.hpp

```
@@ -164,4 +168,12 @@ namespace Math {
     private:
         std::array<Number, static_cast<size_t>(size)> m_cfs;   // Coefficients vector (in increasing degree)
     };
+
+
+
+    /* Returns the sign of a number. */
+    template<typename T>
+    inline int sign(T val) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)